src/SFML/Window/FreeBSD/JoystickImpl.hpp

```diff
@@ -85,15 +85,15 @@ class JoystickImpl
     /// \return Joystick capabilities
     ///
     ////////////////////////////////////////////////////////////
-    JoystickCaps getCapabilities() const;
+    [[nodiscard]] JoystickCaps getCapabilities() const;
 
     ////////////////////////////////////////////////////////////
     /// \brief Get the joystick identification
     ///
     /// \return Joystick identification
     ///
     ////////////////////////////////////////////////////////////
-    Joystick::Identification getIdentification() const;
+    [[nodiscard]] Joystick::Identification getIdentification() const;
```

Comment:
If we ran clang-tidy on the FreeBSD, NetBSD, and OpenBSD codepaths then these changes would have been fixed in #3137 but we don't so I had to manually add them here.